tests/cpp/multidevice.cpp

```diff
@@ -21,6 +21,11 @@
 namespace nvfuser {
 
 MultiDeviceTest::MultiDeviceTest() {
+  // Enable logging. Print debug messagees when knobs like
```

Comment:
Should this be enabled for all tests?

Reply:
Call c10::initLogging in NVFuserTest and leave c10d::setDebugLevelFromEnvironment here? (That would make sense to me.)

Reply:
I was thinking moving both to NVFuserTest. Is there any reason that you may want to do `setDebugLevelFromEnvironment` only with the multi device tests?

Reply:
I see. I missed `d` of `c10d`. Makes sense.